app/controllers/organisations_controller.rb

```diff
@@ -37,14 +37,23 @@ def show
     @can_propose_edits = current_user.present? && !@editable
     @markers = build_map_markers(organisations)
     @cat_name_ids = Category.name_and_id_for_what_who_and_how
+    set_meta_tags title: 'org.name',
```

Comment:
@tansaku yes I just used org.name because that was what was in the show page to keep it consistent, but is as far as I got.